Explorer/package.json

```diff
@@ -37,6 +37,7 @@
     "eslint-config-next": "13.4.4",
     "eslint-config-prettier": "^4.2.1",
     "eslint-plugin-cypress": "^2.13.3",
-    "eslint-plugin-prettier": "^4.2.1"
+    "eslint-plugin-prettier": "^4.2.1",
+    "zustand": "^4.3.8"
```

Comment:
I don't think this should be a dev dependency, as we need it in the final artifact, right?